### ISSUE TITLE
[Feedback] Awesome Plugin

### ISSUE THREAD

**Issue by StefanoRausch** [ivan-lednev/better-search-views#1]
Excellent implementation, and keep up the hard work!

I am beta-testing your plugin and will come back to you with my findings.

**Comment by Azeirah** [comment]
Agreed, Obsidian's default search pane feels so incredibly cluttered and tight. This looks so much nicer!

**Comment by OttoVanluchene** [comment]
There is much more potential with the search. 
People seem to be focusing on the Tags vs Links vs Folders meanwhile you have full text search to find related notes.
Query Control Plugin was such a great plugin, as a replacement for dataview (if you don't need tables)

**Comment by nothingislost** [comment]
Yeah, great job on this. Glad to see more people improving Obsidian's search functionality! Happy to see that my code provided some useful pointers.

**Comment by ivan-lednev** [comment]
@nothingislost, thank you! Hearing this from one of the coolest plugin devs means a lot to me! ❤️
